src/psyclone/tests/domain/common/transformations/kernel_module_inline_trans_test.py

```diff
@@ -1387,6 +1387,14 @@ def test_mod_inline_shared_wildcard_import(monkeypatch, fortran_reader):
     contains
       subroutine do_it()
         real, dimension(10) :: a
+        ! When finding the routine to inline, the frontend does *not* chase the
+        ! imports and therefore eps20 is Unresolved. However, since it must be
+        ! coming from one of the wildcard imports, those wildcard imports are
+        ! moved down into the routine scope. This is watertight for module
+        ! inlining.
+        ! However, if we subsequently want to inline the routine then this
+        ! becomes a problem that could be resolved by adding a flag to try to
+        ! resolve all symbols.
```

Comment:
@arporter As we discussed, I would test is https://github.com/stfc/PSyclone/pull/3211 is sufficient

Reply:
It is :-)